jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/TravelerMessageFromHumanToAsnConverter.java

```diff
@@ -651,7 +634,7 @@ private static ArrayNode transformNodeSetLL(JsonNode nodes) {
     return outputNodeList;
   }
 
-  private static ObjectNode transformNodeLL(JsonNode oldNode) {
+  public static ObjectNode transformNodeLL(JsonNode oldNode) throws InvalidNodeLatLonOffsetException {

```

Comment:
issue: this method is not intended for use outside of this class. As such, it should not be marked as `public`.

suggestion(blocking): since we're testing this method directly (good choice, by the way), we can mark it as `protected` to allow for tests to access the method, while restricting access to this method.
```suggestion
  protected static ObjectNode transformNodeLL(JsonNode oldNode) throws InvalidNodeLatLonOffsetException {

```

Reply:
Makes sense, this in now `protected`!